executor/runtime/docker/docker.go

```diff
@@ -638,6 +655,26 @@ func vpcToolPath() string {
 	return ret
 }
 
+// Use image labels to determine if the container should be configured to run SystemD
+func setSystemdRunning(imageInfo types.ImageInspect, c *runtimeTypes.Container) error {
+	l := log.WithField("imageName", c.QualifiedImageName())
+
+	if systemdBool, ok := imageInfo.Config.Labels[systemdImageLabel]; ok {
+		val, err := strconv.ParseBool(systemdBool)
+		if err != nil {
+			return err
```

Comment:
wrap this error so we know where it's coming from (IIRC, the function is `errors.wrapf`